templates/default/src/main/java/dev/appsody/starter/StarterResource.java

```diff
@@ -1,3 +1,18 @@
+/*******************************************************************************
+Copyright (c) 2020 IBM Corporation and others
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*******************************************************************************/
```

Comment:
Feedback from Graham has been to drop this class (move to samples)... let's only use the Application class to exercise this.

Reply:
ok